tests/test_matrix.py

```diff
@@ -346,6 +346,12 @@ def test_heatmap_annotation_other_data(self):
             assert text.get_text() == f"{val:.1f}"
             assert text.get_fontsize() == 14
 
+    def test_heatmap_annotation_different_shapes(self):
+
+        annot_data = self.df_norm.append(self.letters)
```

Comment:
`DataFrame.append` is a deprecaed method: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.append.html

Please use a different way to construct the annotation matrix (I suggest `self.df_norm.iloc[:-1]`)